ingest/dense.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, file_path, file_id, study_accession, file_params):
         self.file_id = file_id
         self.study_accession = study_accession
         self.cell_names = self.get_next_line(increase_line_count=False)[1:]
-        self.file_params = file_params
+        self.matrix_params = {k: v for k, v in kwargs.items() if v is not None}
```

Comment:
Helpful -- I'd suggest adding that comment directly in the code.

```suggestion
        # Remove from dictionary any keys that have value=None
        self.matrix_params = {k: v for k, v in kwargs.items() if v is not None}
```